web/frontend/src/pages/form/IndividualResult.tsx

```diff
@@ -62,82 +62,89 @@ const IndividualResult: FC<IndividualResultProps> = ({
     OUT_OF_BOUNDS = 2,
   }
 
-  const questionIcons = {
-    [RANK]: <SwitchVerticalIcon />,
-    [SELECT]: <CursorClickIcon />,
-    [TEXT]: <MenuAlt1Icon />,
-  };
-  const SubjectElementResultDisplay = (element: SubjectElement) => {
```

Comment:
See https://dev.to/amitaldo/function-makes-the-dependencies-of-useeffect-hook-change-on-every-render-warning-in-react-35hd